frontend/admin/app/Http/Controllers/AuthController.php

```diff
@@ -61,6 +69,32 @@ public function authCallback(Request $request)
             'redirect_uri' => config('oauth.redirect_uri'),
             'code' => $request->code,
         ]);
+
+        // decode id_token here
+        // pull token out of the response
+        // problem, idToken is null
+        // problem input() and query() are undefined methods for $response
+        $idToken = $response->input('id_token');
```

Comment:
```suggestion
        $idToken = $response->json('id_token');
```
Sorry, I sent you in the wrong direction with `input`.  The token is actually in the json payload.  You can pull it out like this.  Remember, it will be null for Laravel auth.